pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=41.0.1", "Cython>=3.0.0a11"]
+requires = ["setuptools>=41.0.1", "Cython>=0.29.24,<=3.0.0a11"]
```

Comment:
For Python 3.12 compatibility, should this use the same environment markers you wrote in `requirements/build.txt`? (They are supported here: see [PEP 518](https://peps.python.org/pep-0518/#build-system-table), [PEP 508](https://peps.python.org/pep-0508/#environment-markers).)

Reply:
Yes, that would be great.

Reply:
TIL, as well.  Repushed with those.